unittests/CppInterOp/ScopeReflectionTest.cpp

```diff
@@ -671,7 +671,10 @@ TEST(ScopeReflectionTest, GetBaseClassOffset) {
   class F : public A { int m_f; };                              \
   class G : public F { int m_g; };
 
-  CODE;
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-private-field"
+CODE;
+#pragma GCC diagnostic pop
```

Comment:
Let's drop the pragmas and make classes `struct`s. This way the `m_*` will become public and will make the warning go away.

Reply:
That will fix most warnings, but this won't fix the warning about   const int N = 5; being unused VariableReflectionTest.cpp.

Reply:
https://github.com/compiler-research/CppInterOp/actions/runs/9142202412/job/25137488586?pr=288#step:15:185 Link to error with const int N = 5; Unless there is a specific reason its being labelled const then I can try fixing it by removing the const?

Reply:
Testing removing const to fix. Will revert and try something else if needed